src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -4,19 +4,20 @@
 import java.util.function.Predicate;
 
 import seedu.address.commons.util.StringUtil;
+import seedu.address.model.residence.Booking;
```

Comment:
do you mean seedu.address.model.booking.Booking? I think you might have forgotten to refactor the person package to 'booking' instead?

Once you refactor, you probably don't need this import statement since the Booking is also in the same package and you can use it directly in this file

Reply:
@VRSoorya So booking should be its own package under model? Because I recall someone saying booking should be under residence, but I can have it in its own package if that works better